iodata/test/test_chgcar.py

```diff
@@ -21,56 +21,62 @@
 # pragma pylint: disable=invalid-name,no-member
 """Test iodata.chgcar module."""
 
-
 import numpy as np
+from numpy.testing import assert_equal, assert_allclose
+
+from ..utils import angstrom, volume
+from ..iodata import IOData
+from ..chgcar import _unravel_counter
 
-from .. utils import angstrom, volume
-from .. iodata import IOData
-from .. chgcar import _unravel_counter
 try:
     from importlib_resources import path
 except ImportError:
     from importlib.resources import path
 
 
 def test_unravel_counter():
-    assert _unravel_counter(0, [3, 3, 3]) == [0, 0, 0]
-    assert _unravel_counter(0, [2, 4, 3]) == [0, 0, 0]
-    assert _unravel_counter(1, [2, 4, 3]) == [1, 0, 0]
-    assert _unravel_counter(2, [2, 4, 3]) == [0, 1, 0]
-    assert _unravel_counter(3, [2, 4, 3]) == [1, 1, 0]
-    assert _unravel_counter(8, [2, 4, 3]) == [0, 0, 1]
-    assert _unravel_counter(9, [2, 4, 3]) == [1, 0, 1]
-    assert _unravel_counter(11, [2, 4, 3]) == [1, 1, 1]
-    assert _unravel_counter(24, [2, 4, 3]) == [0, 0, 0]
+    assert_equal(_unravel_counter(0, [3, 3, 3]), [0, 0, 0])
+    assert_equal(_unravel_counter(0, [2, 4, 3]), [0, 0, 0])
+    assert_equal(_unravel_counter(1, [2, 4, 3]), [1, 0, 0])
+    assert_equal(_unravel_counter(2, [2, 4, 3]), [0, 1, 0])
+    assert_equal(_unravel_counter(3, [2, 4, 3]), [1, 1, 0])
+    assert_equal(_unravel_counter(8, [2, 4, 3]), [0, 0, 1])
+    assert_equal(_unravel_counter(9, [2, 4, 3]), [1, 0, 1])
+    assert_equal(_unravel_counter(11, [2, 4, 3]), [1, 1, 1])
+    assert_equal(_unravel_counter(24, [2, 4, 3]), [0, 0, 0])
 
 
 def test_load_chgcar_oxygen():
     with path('iodata.test.data', 'CHGCAR.oxygen') as fn:
         mol = IOData.from_file(str(fn))
-    assert (mol.numbers == [8]).all()
-    assert abs(volume(mol.rvecs) - (10 * angstrom) ** 3) < 1e-10
+    assert_equal(mol.numbers, 8)
+    assert_allclose(volume(mol.rvecs), (10 * angstrom) ** 3, atol=1.e-10)
     ugrid = mol.grid
-    assert len(ugrid['shape']) == 3
-    assert (ugrid['shape'] == 2).all()
-    assert abs(ugrid['grid_rvecs'] - mol.rvecs / 2).max() < 1e-10
+    assert_equal(len(ugrid['shape']), 3)
+    assert_equal(ugrid['shape'], 2)
```

Comment:
Very minor comment and I'm only mentioning this because it could be useful to know. The above two lines can be written as:
```python
assert_equal(ugrid['shape'], [2, 2, 2])
```

Reply:
It's good to know for its simplicity. Thank you.